### DISCUSSION TITLE
Lecture 3 : Not able to transfer funds, something with address payable

### DISCUSSION THREAD

**Discussion by Noveleader** [smartcontractkit/full-blockchain-solidity-course-py#1221] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/91677627/157244741-5d975810-36b7-4302-9a7d-98fb86af528e.png)

**Comment by cromewar** [comment]
Hello @Noveleader 
This is strange  as your code seems to be ok, which compiler version are you using?

**Reply by Noveleader** [reply]
@cromewar 
I am using following version:
pragma solidity >=0.6.6 <0.9.0;

**Reply by cromewar** [reply]
For me is most likely an error with the cache of your browser, but not 100% sure. Could you please try the exact same code in another browser?

**Reply by Noveleader** [reply]
Ok, let me try

**Reply by Noveleader** [reply]
Hey @cromewar I tried what you said but it isn't working

**Reply by cromewar** [reply]
This is weird, I just tried to reproduce the code to that point and it worked just fine, could you share please the compile version (not int the c contract but on remix.)

**Reply by Noveleader** [reply]
Hey @cromewar I didn't get you but this is my code:
https://github.com/Noveleader/Solidity-/blob/main/FundMe.sol

**Reply by Noveleader** [reply]
![image](https://user-images.githubusercontent.com/91677627/157256553-4c5e25aa-a536-4e27-bb1b-0b1b26646ddd.png)

**Reply by Noveleader** [reply]
@cromewar any solutions on this ?

**Reply by cromewar** [reply]
Seem a bug on remix as I cannot find an actual error here.

**Reply by JohnEMagee** [reply]
line 37 has a red exclamation point, it's calling an error.  What is the message when you hover over it, and what does the rest of your code look like?

**Reply by Noveleader** [reply]
Hey @JohnEMagee this is my full code:
https://github.com/Noveleader/Solidity-/blob/main/FundMe.sol

**Reply by Noveleader** [reply]
> Seem a bug on remix as I cannot find an actual error here.

Ok

**Comment by Noveleader** [comment, marked as answer]
Actually, in the newer versions of solidity to send some money to the address, we need to make the address payable
Hence the correct code is following here:
`payable(msg.sender).transfer(address(this).balance);`

**Reply by cromewar** [reply]
Incredible catch @Noveleader, good work!!

**Reply by Noveleader** [reply]
Thanks
